chart/templates/agent-smith-daemonset.yaml

```diff
@@ -28,6 +28,7 @@ spec:
         component: agent-smith
         kind: daemonset
         stage: {{ .Values.installation.stage }}
+      name: agent-smith
```

Comment:
This is just a cheap change to make it more identical to the installer ouput. And it does no harm.